### DISCUSSION TITLE
Add UI Page for Indexer Configuration

### DISCUSSION THREAD

**Discussion by slemens** [pymedusa/Medusa#7570] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
I want to add a new local indexer, that is complete TVDBv2 compatible. Reason for this is that I want to use local air dates. Other countrys than US waste their API hits to newnabb tracker when trying to search for episodes that are not released in their language/country. 
Within sickhill i could add the local indexer with to changes to indexer_config.py and tvdb_api.py but I could'nt find a way to do the same within medusa.

**Describe the solution you'd like**
I'd like to have a UI page where we can configure new indexer, or a central config file to define the indexer.

**Describe alternatives you've considered**
Alternative could be a hint from the developers, where I have to change the sources to get a new indexer.

**Additional context**
Attached you could see some screenshots from the API documentation of our local indexer. The site is not public accessable, so sorry for only posting screenshots.

![glotz1](https://user-images.githubusercontent.com/25345664/72201788-d810cc00-3457-11ea-865d-6d7551e60136.PNG)
![glotz2](https://user-images.githubusercontent.com/25345664/72201791-da732600-3457-11ea-953f-4bc04c87bd47.PNG)

**Comment by p0psicles** [comment]
Wait. So you have created your own personal indexer, like a complete copy of tvdb apiv3, just to modify the airdates? So you can save some api hits? I don't get it

**Comment by slemens** [comment]
Kind of, but it has been created by a huge german community named glotz.info. It is a huge improvement for us, because german indexer are really rare and the have a very limited number of daily API hits.
Think of it as a kind of proxy to the TVDB API that manipulates air dates and networks to match our local air dates.

**Comment by p0psicles** [comment]
First check out the airdate offset feature. Maybe that helps out your usecase. It's in the edit show - advanced section

**Comment by slemens** [comment]
Thank you for you answer. I currently manage 173 shows through Sickchill and want to switch to medusa. I've made a dry run and my backlog has more than 1200 episodes because of the wrong airdates. The iondexer I use allows 1000 API hits a day ;). To correct them manually for each show is not really a option for me.

**Comment by p0psicles** [comment]
You can always add an indexer in your own fork. But if it's not usable for everyone, it will never get merged.

I started on adding IMDb as an indexer. But never finished it. You could take a look in the branch to see what's needed to add an indexer. https://github.com/pymedusa/Medusa/pull/3603

Copy tvdb into a new folder and use the id 11.
